.github/workflows/format.yml

```diff
@@ -0,0 +1,38 @@
+name: Format Code
+on:
+  pull_request:
+    branches: [main]
```

Comment:
This isn't a critical action, so I just run it on PRs to main.

Reply:
Love the idea of auto-formatting on PR (as opposed to just rejecting it). Asking to run formatting is a fairly common comment on PRs, so this saves some back and forth and keeps the codebase clean